consul-core/src/test/java/com/smoketurner/dropwizard/consul/core/ConsulAdvertiserTest.java

```diff
@@ -49,6 +49,7 @@
   public static final String SECOND_SUBNET_IP = "192.168.2.99";
   public static final String FIRST_SUBNET_IP = "192.168.1.53";
   public static final String THIRD_SUBNET_IP = "192.168.3.32";
+  private static final String DEFAULT_HEALTH_CHECK_PATH = "ping";
```

Comment:
can you change the default to "healthcheck" to preserve the existing behavior?

Reply:
Sure can, I have updated it to use "healthcheck" as the default.